Foxtrot/Foxtrot/DuplicatorForContractsAndClosures.cs

```diff
@@ -72,6 +72,15 @@ internal class DuplicatorForContractsAndClosures : Duplicator
                         dup.DuplicateFor[sourceMethod.Parameters[i].UniqueKey] = targetMethod.Parameters[i];
                     }
                 }
+
+                if (sourceMethod.TemplateParameters != null && targetMethod.TemplateParameters != null
```

Comment:
The only suggestion is to put a comment that will explain (very shortly) what problem this code is trying to solve.